docs/ref/utils.txt

```diff
@@ -1077,6 +1077,12 @@ For a complete discussion on the usage of the following see the
 
     ``override`` is also usable as a function decorator.
 
+.. function:: check_for_language(lang_code)
+
+    Checks whether there is a global language file for the given language
+    code (e.g. 'fr', pt_BR', …). This is used to decide whether a user-provided
```

Comment:
'pt_BR'
I think the three dots could be omitted, or including them as 3 separate characters instead of the single character version seems to be more common.